parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -125,10 +125,10 @@ installSGXDrivers() {
     VERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
     case $VERSION in
     "18.04")
-        SGX_DRIVER_URL="https://download.01.org/intel-sgx/dcap-1.2/linux/dcap_installers/ubuntuServer18.04/sgx_linux_x64_driver_1.12_c110012.bin"
+        SGX_DRIVER_URL="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer18.04/sgx_linux_x64_driver_1.21.bin"
```

Comment:
Is this URL going to break when there's a new `sgx_linux` driver release? I'm guessing from the contents of https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer18.04/ that the 1.22 release (for example) would replace this one,

Reply:
@mboersma you are assuming based on the presence of a single versioned artifact in that directory that only one at a time ever lives there?

Reply:
Yes, although I could be wrong.

Reply:
@yakman2020 can you confirm whether or not this URL will be resilient when the next version drops?

Reply:
Given that SGX drivers installation is currently broken we'll merge this as-is, with the understanding that it may break again in the future